avm/res/desktop-virtualization/application-group/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,54 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-desktopvirtualization.applicationgroup-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
```

Due to a change implemented in #838. Please update all `main.test.bicep` files with the following updated 

```suggestion
param resourceLocation string = deployment().location
```

Please note all subsequent refences to the location parameter will need to be updated as per the following example:
` location: resourceLocation`

---

@ChrisSidebotham - should we maybe wait with this until the CI can support it?